package.json

```
@@ -9,7 +9,7 @@
 		"check": "pnpm -r check",
 		"lint": "pnpm -r lint",
 		"format": "pnpm -r format",
-		"release": "pnpm publish --tag=next --filter=\"@sveltejs/*\" --filter=\"create-svelte\"",
+		"release": "pnpm publish --tag=next --filter=\"@sveltejs/*\" --filter=\"create-svelte\" && git push --follow-tags",
```

is this the right change @antony / @pngwn?

```suggestion
		"release": "pnpm build -r && changesets publish && git push --follow-tags",
```

---

I think you call `changeset` via pnpm i.e. `pnpm changeset version` rather than directly.

I'll update the PR.

---

Ok done, one question, why the `pnpm build -r`. We have already built the monorepo at this point via `pnpm build` which is the previous step in `release.yaml`.

I think we'd be safe to remove it from here.

---

Pretty sure the publish step was building, so I was just copying what was already happening. If that means we've been building twice and now we only need to build once, then great